backend/src/v2/objectstore/object_store.go

```diff
@@ -188,7 +188,7 @@ func ParseBucketConfig(path string) (*Config, error) {
 	}
 
 	// TODO: Verify/add support for file:///.
-	if ms[1] != "gs://" && ms[1] != "s3://" && ms[1] != "minio://" {
+	if ms[1] != "gs://" && ms[1] != "s3://" && ms[1] != "minio://" && ms[1] != "mem://" {
```

Comment:
`mem://` to support in memory storage for launcher tests under environments that doesn't have access to public gcs endpoints